src/workflows-app/WorkflowsInWorkspace.ts

```diff
@@ -62,6 +67,15 @@ export const WorkflowsInWorkspace = ({
     [signal, workspaceId]
   );
 
+  const deleteMethod = useCallback(
+    async (methodId) => {
+      const { cbasProxyUrlState } = await loadAppUrls(workspaceId, 'cbasProxyUrlState');
```

Comment:
Is this already in the page context or going out to the network again?

Reply:
good call - we can get what we need from page context

Reply:
I wasn't able to do this easily -- if any reviewer feels this is really important and is willing to help me make it work, let me know, otherwise I don't think it's that impactful.